CMakeLists.txt

```diff
@@ -145,7 +145,7 @@ math(EXPR GNURADIO_BCD_VERSION
 add_definitions(-DGNURADIO_VERSION=${GNURADIO_BCD_VERSION})
 
 if(Gnuradio_VERSION VERSION_LESS "3.8")
-    find_package(Boost COMPONENTS system REQUIRED)
+    find_package(Boost COMPONENTS system thread REQUIRED)
```

Comment:
Could you please use `std::thread` instead? I'm trying to remove all usage of Boost from Gqrx.

Reply:
Thank you for suggestion.
Done in 2db43829817932406037598e8a3dadf654c3346f.
boost::thread dependency removed in fce4a22876f7a7b6efee38053ad0bfb2d181fcc1.